HTAN.model.csv

```diff
@@ -775,4 +775,25 @@ Phospho Site,The protein site for a phosphoprotein targeting antibody. Report AA
+PMID,The PubMed identifier associated with the publication (applicable to published manuscripts),,,,FALSE,Publication,,,
+Data Type,"Types of data associated with the content. Fill out Other Data Type Specified, if not on the list.","10X Visium, Bulk DNA, Bulk RNA-seq, CyCIF, H&E, IMC, LC-MS-MS, LC-MS3, MIBI, MxIF, Shotgun MS (lipidomics), mIHC, scATAC-seq, scRNA-seq, t-CyCIF, Demographics",,,TRUE,Publication,,,list

```

Comment:
Is this the list of all valid data types (union of assay type and imaging assay type) or the list of assay types received to date. If the latter we should update to the format.

Reply:
Have added in all valid imaging assay types to valid values here. Interestingly plain `assay type` does not have set valid values.